packages/bpk-tokens/src/android/base/aliases.json

```diff
@@ -41,7 +41,9 @@
     "BORDER_SIZE_SM": 1,
     "BORDER_SIZE_LG": 2,
     "BORDER_SIZE_XL": 3,
+    "BORDER_RADIUS_XS": 4,
     "BORDER_RADIUS_SM": 4,
```

Comment:
This value has been kept for now until the work in the platform to use is in progress so that if any urgent work comes up we aren't forced to adopt the breaking change of moving this to `6`